packages/swingset-liveslots/test/test-initial-vrefs.js

```diff
@@ -91,7 +91,8 @@ test('initial vatstore contents', async t => {
   t.deepEqual(kunser(JSON.parse(get(`vc.2.|schemata`))), scalarSchema);
   t.deepEqual(kunser(JSON.parse(get(`vc.3.|schemata`))), scalarSchema);
   // watchedPromises uses vref (string) keys
-  t.deepEqual(kunser(JSON.parse(get(`vc.4.|schemata`))), stringSchema);
+  const scalarStringSchema = [M.and(M.scalar(), M.string())];
+  t.deepEqual(kunser(JSON.parse(get(`vc.4.|schemata`))), scalarStringSchema);
```

Comment:
@FUDCo I "fixed" this test so that it passes, without really understanding what's going on. Did I change the test correctly?

Reply:
Per our video conversation, yup.

Reply:
Thanks. Resolving.